etc/sync-wip-spec.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
```

Comment:
did you mean to add this file?

Reply:
I have a note in the PR description, I intend to remove it when the spec merges. We can keep this thread to make sure it does get removed

Reply:
Removed